x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/connector_configuration.tsx

```diff
@@ -117,41 +118,21 @@ export const ConnectorConfiguration: React.FC = () => {
                 {
                   children: (
                     <>
-                      <EuiText size="s">
-                        <FormattedMessage
-                          id="xpack.enterpriseSearch.content.connector_detail.configurationConnector.connectorPackage.description.secondParagraph"
-                          defaultMessage="The connectors repository contains several {link}. Use our framework to accelerate developing connectors for custom data sources."
-                          values={{
-                            link: (
-                              <EuiLink
-                                href="https://github.com/elastic/connectors-python/tree/main/connectors"
-                                target="_blank"
-                                external
-                              >
-                                {i18n.translate(
-                                  'xpack.enterpriseSearch.content.connector_detail.configurationConnector.connectorPackage.clientExamplesLink',
-                                  { defaultMessage: 'connector client examples' }
-                                )}
-                              </EuiLink>
-                            ),
-                          }}
-                        />
-                      </EuiText>
                       <EuiSpacer />
                       <EuiText size="s">
                         <FormattedMessage
                           id="xpack.enterpriseSearch.content.connector_detail.configurationConnector.connectorPackage.description.thirdParagraph"
-                          defaultMessage="In this step, you will need to clone or fork the repository, and copy the generated API key and connector ID to the associated {link}. The connector ID will identify this connector to Search. The service type will determine which type of data source the connector is configured for."
+                          defaultMessage="In this step, you will need to clone or fork the elastic/connectors repository, and copy the API key and connector ID values to the config.yml file. Here's an {exampleLink}."
                           values={{
-                            link: (
+                            exampleLink: (
                               <EuiLink
-                                href="https://github.com/elastic/connectors-python/blob/main/config.yml"
+                                href="https://github.com/elastic/connectors-python/blob/main/config.yml.example"
```

Comment:
@seanstory one stray substitution we overlooked was...Kibana! 😄

Reply:
🙈 Good catch! Thanks!

Reply:
Does stink though that this link will be broken for all <8.13 kibanas in the world 😬